func/omp/create-parallel.cpp

```diff
@@ -0,0 +1,22 @@
+#include <omp.h>
+#include <stdio.h>
+#include <faasm/faasm.h>
+
+void printThreadInfo(const char* label) {
```

Comment:
When adding a new function to the codebase we should check in the `.wasm` file as well as the source. 

In this diff it looks like `hellomp.wasm` has changed but `create_parallel.wasm` hasn't been added.